src/components/FilterEditorWindow/config/RequestFromCode.js

```diff
@@ -69,8 +77,23 @@ async function discoverFromCode(collectionName, data) {
     };
   }
 
+  // Then score these random points
+  const idsToInclude = randomResponse.data.result.points.map((point) => point.id);
+  const mustFilter = [{ has_id: idsToInclude }];
+  data.reqBody.filter = originalFilter || {};
+  data.reqBody.filter.must = mustFilter.concat(data.reqBody.filter?.must || []);
+
+
+  const scoredRandomResponse = await actionFromCode(collectionName, data, 'discover');
+  if (randomResponse.error) {
+    return {
+      data: null,
+      error: randomResponse.error,
+    };
+  }
```

Comment:
I guess this was copy/past :)
```suggestion
  const scoredRandomResponse = await actionFromCode(collectionName, data, 'discover');
  if (scoredRandomResponse.error) {
    return {
      data: null,
      error: scoredRandomResponse.error,
    };
  }
```